frontend/components/forms/fields/InputField/_styles.scss

```diff
@@ -10,6 +10,7 @@
   box-sizing: border-box;
   height: 40px;
   transition: border-color 100ms;
+  width: 100%;
```

Comment:
good PR but this one line concerns me a bit as its changes the input width everywhere. Lets just be sure we are not creating any UI bugs from this. other wise looks good.

Reply:
@ghernandez345 sounds good – I'll give this another pass for any UI bugs

Reply:
@ghernandez345 I looked through many of the places where InputField is used and on disabling/re-enabling `width: 100%` none of them seemed to change (so they were behaving that way already). I think it's okay, but am not sure – ideally QA would catch anything I miss right? Do you think I should merge this or make it a more local width rule?

Reply:
if we've checked and feel confident we wont add regression that its fine. i'd just merge in as is.